ReactSkia/components/RSkComponent.h

```diff
@@ -23,6 +24,14 @@ enum ComponentUpdateMask {
       ComponentUpdateMaskState | ComponentUpdateMaskLayoutMetrics
 };
 
+struct props{
+    SkColor backgroundColor;
+    SkColor foregroundColor;
+    int pointerEvents;
+    EdgeInsets hitSlop;
+    bool masksToBounds_;
```

Comment:
masksToBounds_ here '_' is used for private members . is it required for this member ?

Reply:
yes, it is required this member for overflow.